examples/minimum_blending.py

```diff
@@ -0,0 +1,41 @@
+"""
+Minimum blending
+================
+
+Demonstrates how to use the `minimum` blending mode with inverted colormaps.
+`minimum` blending uses the minimum value of each R, G, B channel for each pixel.
+`minimum` blending can be used to yield multichannel color images on a white 
+background, when the channels have inverted colormaps assigned.
+An inverted colormap is one where white [1, 1, 1] is used to represent the lowest 
+values, as opposed to the more conventional black [0, 0, 0].
+"""
+
+import numpy as np
+from vispy.color import Colormap
+from skimage import data
+import napari
+
+# First, generate two inverted colormaps that go from white [1, 1, 1] 
+# to the chosen color.
+# These are from ChrisLUTs: https://github.com/cleterrier/ChrisLUTs
+# and correspond to the `green` and `magenta` colormaps used by default
+# when opening this sample from the File menu.
```

Comment:
@andy-sweet Same as above, should I add mention of the new ones added to the GUI? Or just delete this block and use a pair of the new ones (say Purple and Orange).

Reply:
As we're adding them, I think you should use of the new ones and remove the comment block.